lib/upgradedb/postgres.2022072200.php

```diff
@@ -23,8 +23,14 @@
 
 $this->BeginTrans();
 
-$this->Execute("ALTER TABLE cashsources ADD COLUMN isdefault smallint NOT NULL DEFAULT 0");
+$this->Execute("ALTER TABLE events ADD COLUMN netnodeid integer");
```

Comment:
Brak `DEFAULT NULL`.

Reply:
To co wyżej - zrobię jak chcesz tylko odpowiedz na pytanie wyżej https://github.com/chilek/lms/pull/2273#discussion_r931247474

Reply:
https://github.com/chilek/lms/pull/2273/commits/332668c2474b4eae71f096c09e7691846d5cf380